for_loop_practice.py

```diff
@@ -0,0 +1,58 @@
+"""
+author: kalez43@github.com
+description: small practice for loops
+"""
+# lists of questions and their respective answers
+questions = ["What is 1 + 1?", "What is 2 + 2?", "What is 3 * 3?"]
+answers = [2, 4, 9]
+
+# for loop asking each question
+for current in range(0, len(questions)):
+    current_question = questions[current]  # assigns current question so we can compare later
+    current_answer = answers[current]  # assigns current answer so we can compare later
+    print(questions[current])  # prints current question
+    user_answer = int(input("Type answer and press enter ==> "))  # asks user to answer current question
+    if user_answer == current_answer:  # evaluates whether user input is correct
+        print("Well done!")  # prints success message
+        continue  # continues the loop
+    else:
+        print("Sorry, that is incorrect.")  # prints failure message
+        break  # stops the loop
+
+
+# creates function that prints a row of stars
+def stars():
+    print('***')
+
+
+stars()  # prints row of stars
+
+# lists
+letters = "three"
+numbers = [0, 1, 2, letters, 4, 5, 6]
+
+
+# creates function for printing each item in the list 'numbers'
+def print_list(lst):
+    for num in lst:
+        print(num)
+
+
+print_list(numbers)  # prints the list 'numbers'
+
+stars()  # prints row of stars
+
+# list
+colors = ["green", "blue", "pink", "yellow", "orange"]
+
+
+# function that runs a for loop to print an item of a list if it matches item position of n
+def print_color(color_lst):
+    for color in color_lst:

```

Comment:
Why does it have to be inside the loop? and why are you reading from colors and not color_lst?

Reply:
You're right, colors should be changed to color_lst. This function is not really very useful, it was just to help me practice writing for loops and if statements and nesting them inside of each other.